Makefile

```diff
@@ -4,12 +4,12 @@ ifneq (,$(wildcard ./.env))
 else
   $(error You haven't setup your .env file. Please refer to the readme)
 endif
-GOOS=linux
-GOARCH=arm64
-GOCC?=go
-GOFLAGS=-tags=lambda.norpc
-CGO_ENABLED=0
-LAMBDAS=build/getclaims/bootstrap build/getroot/bootstrap build/notifier/bootstrap build/postclaims/bootstrap build/providercache/bootstrap build/remotesync/bootstrap
+LAMBDA_GOOS=linux
+LAMBDA_GOARCH=arm64
+LAMBDA_GOCC?=go
+LAMBDA_GOFLAGS=-tags=lambda.norpc
+LAMBDA_CGO_ENABLED=0
```

Comment:
Overwriting these env vars causes problems when building for local execution i.e. `ucankey`

Reply:
yeah, that makes sense as they are the vars Go uses